xc/device.ml

```diff
@@ -2066,6 +2066,7 @@ module Backend = struct
       let suspend (task: Xenops_task.task_handle) ~xs ~qemu_domid domid =
         Dm_Common.suspend task ~xs ~qemu_domid domid;
         let file = sprintf qemu_save_path domid in
+        qmp_write domid (Qmp.Command(None, Qmp.Stop));
```

Comment:
How about this - and also changing the line below?
```
Qmp.(Command(None, Stop));
```

Reply:
that would be nice, I've added a new commit: https://github.com/xapi-project/xenopsd/pull/415/commits/963c41ef6ccee63d3b0e3c4ba9dd0140efc3d9b9